src/lib/pages/account-details/components/tables/transactions/index.tsx

```diff
@@ -0,0 +1,199 @@
+import { Box, Flex, Grid } from "@chakra-ui/react";
+import type { ChangeEvent } from "react";
+import { useMemo, useState } from "react";
+
+import { Loading } from "lib/components/Loading";
+import { Pagination } from "lib/components/pagination";
+import { usePaginator } from "lib/components/pagination/usePaginator";
+import { EmptyState } from "lib/components/state/EmptyState";
+import { TableContainer, TableHeader } from "lib/components/table";
+import { TableTitle } from "lib/components/table/TableTitle";
+import { ViewMore } from "lib/components/table/ViewMore";
+import { FilterSelection } from "lib/pages/past-txs/components/FilterSelection";
+import { useTxQuery } from "lib/pages/past-txs/query/useTxQuery";
```

Comment:
@evilpeach should we move these components from past-txs to a common directory?

Reply:
I think we just add comment `//revist: refactor later` for now.